hammer/library/jiraoperations.py

```diff
@@ -266,6 +268,27 @@ def assign_user(self, ticket_id, assinee_name):
         logging.debug(f"Assigned {ticket_id} to {assinee_name}")
         return True
 
+    def add_label(self, ticket_id, label):
+        """
+                add label to `ticket_id`.
+
+                :return: boolean, if label update was successful
+                """
+        if not (ticket_id or label):
+            return False
+
+        try:
+            issue = self.session.issue(ticket_id)
+            issue.fields.labels.append(label)
+            issue.update(fields={"labels": issue.fields.labels})
```

Comment:
Will it try to add new label if such label is already exists?

Reply:
it will overwrites the label if already exists. if not add new label.